Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RumbleBindingSetting.java

```diff
@@ -46,8 +47,8 @@ private void saveRumble(Settings settings, InputDevice device)
     }
     else
     {
-      setValue(settings, "",
-              DolphinApplication.getAppContext().getString(R.string.rumble_not_found));
+      Toast.makeText(DolphinApplication.getAppContext(), R.string.rumble_not_found,
+              Toast.LENGTH_SHORT).show();
```

Comment:
Was changing this from saving a setting to showing a toast intentional?

Reply:
Yes. I don't see a reason to save this string since it would be formatted differently than the rest of the binding settings. I think it makes more sense to just notify the user if rumble isn't found and not modify any settings.